Makefile

```diff
@@ -7,7 +7,7 @@ GOPATH:=$(shell go env GOPATH)
 
 VERSION?=latest
 
-GOLANGCI_VERSION:=1.55.2
+GOLANGCI_VERSION:=1.63.4
```

Comment:
It looks good to me. 
Just out of curiosity, is it the mandatory thing to bump golangci-lint?

Reply:
Yes, it was throwing all sorts of errors in previous ci runs.

Reply:
Thanks, got it.